recommendations/test_sms_parser.py

```diff
@@ -168,7 +162,6 @@ def test_parse_is_invite(self):
         self.assertEqual(result['action'], ACTIONS_LIST['invite'])
         self.assertEqual(result['payload']['invite_number'], invite_number)
 
-    @skip('known issue #50')
```

Comment:
Because regex is "smarter" than our old parser, we fixed these issues in the process of implementing it.